code/modules/materials/material_recipes.dm

```diff
@@ -17,10 +17,10 @@
 	if(holographic || phase_at_temperature() != MAT_PHASE_SOLID)
 		return list()
 
-	// By default we don't let anything be crafted with ore, as it's too raw.
+	// By default we don't let anything be crafted with ore or logs, as they are too raw.
 	// We make an exception for clay as it is being moulded by hand.
 	. = list()
-	if(ispath(stack_type, /obj/item/stack/material/ore))
+	if(ispath(stack_type, /obj/item/stack/material/ore) || ispath(stack_type, /obj/item/stack/material/log))
```

Comment:
we already have brick walls, i'm unsure if adding log walls is out of scope but it'd make sense. admittedly it might not make sense without either cutting wedges into the logs so they'll stack without rolling off, or having some other kind of binding material like ropes or rags or mud or clay, and i don't *think* we have support for either of those when crafting